src/page/menu-content.page.ts

```diff
@@ -4,7 +4,7 @@ export class MenuContentPage {
   private tShirtMenu: ElementFinder;
 
   constructor () {
-    this.tShirtMenu = $('#block_top_menu > ul > li:nth-child(3) > a');
+    this.tShirtMenu = $('.sf-menu > :nth-child(3) > a');
```

Comment:
Use a different selector, this is depending on the position of the T-Shirts option, you can try to use the title attribute and also some of the existing elements in the current locator.